docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
     depends_on:
       - db
   db:
-    image: genomenexus/gn-mongo:latest
+    build: https://github.com/inodb/genome-nexus-importer.git#add-exon-info-to-transcript
```

Comment:
What does this do? Does it just make an image using the Dockerfile here https://github.com/inodb/genome-nexus-importer.git#add-exon-info-to-transcript? Should this be pointing to `github.com/inodb/` ?

Reply:
I think necessary because PR for exon information is not yet merged. When merged this should be changed to `github.com/genome-nexus/genome-nexus-importer.git` I guess.

Reply:
@pieterlukasse yes it makes it using the Dockerfile there. You can even select different dockerfiles in the same repo if required. We should indeed change it to point to main fork once merged as @dionnezaal mentioned